src/utils/externalService.tsx

```diff
@@ -0,0 +1,33 @@
+import { ReactNode } from 'react';
+import { IconEnvelopeOutline, IconGithubOutline, IconPhoneOutline, IconTelegramOutline } from '@taskany/icons';
+
+import { tr } from './utils.i18n';
+
+type ExternalServiceInfo = {
+    name: string;
+    icon: ReactNode;
+    getLink?: (id: string) => string;
+};
+
+export const externalServicesInfo: Partial<Record<string, ExternalServiceInfo>> = {
+    email: {
```

Comment:
it should be a database set of options so users could customize their set

Reply:
What about service icons? I guess we can store svg in db

Reply:
I feel like we can just use our taskany/icons pack and use those? Storing only name